parser/parser.y

```diff
@@ -656,9 +659,25 @@ Symbol:
 
 /*******************************************************************************************/
 Assignment:
-	ColumnName eq Expression
+	QualifiedIdent eq Expression
```

Comment:
QualifiedIdent contains `db.table.columnname` or `*`, you should handle these for error, maybe.